### PR TITLE
[INTERNAL] incorporate docu feedback

### PR DESCRIPTION
**Pull request by florianstephan01**
Documentation Update

### CODE REVIEW

File: doc/IMPLEMENTATION.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 An UI Extension implementation starts with the creation a new UI5 XML View that refers to the <code>sap.i2d.lo.lib.vchclf.api.valuation.controller.CustomCharacteristicGroupBase</code> controller. In the next step Layout Containers are inserted and the [StandardCharacteristic](IMPLEMENTATION.md#the-standardcharacteristic-control) and [CustomCharacteristic](IMPLEMENTATION.md#the-customcharacteristic-control) control are placed and bound to the corresponding Characteristics.
 
+> ### Attention - Namespace adoptions
+> - When the namespace of the template is changed according to own name schemas, it need to be considered that the namespace is represented with <code>.</code> and <code>/</code> notation as separation marker. This depends on the location of the specific defintion.
```
Comment (danielheffter) [inline doc/IMPLEMENTATION.md]: small typo: "..., it need**s**"

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js [hunk 2]
```diff
@@ -8,6 +8,22 @@ sap.ui.define([
 ], function(ManagedObject, formatMessage) {
 	"use strict";
 
+	/**
+     * Implementation of the Characteristic Group Client API. Each VC UI Extension view controller (which needs to be derived from
+	 * {@link module:sap/i2d/lo/lib/vchclf/api/valuation/controller/CustomCharacteristicGroupBase}
+     * ) owns an instance of this object and has access. It allows to retrieve Characteristic data independent of the context to
+	 * which a {@link module:sap/i2d/lo/lib/vchclf/api/valuation/control/CustomCharacteristic}
+	 * is bound. Furthermore loading of Domain Values can be manally triggered (e.g. for an custom Value Help implementation) and
```
Comment (danielheffter) [inline webapp/libs/api/valuation/CharacteristicGroupClientObject.js]: small typo: "manually"

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js [hunk 3]
```diff
@@ -8,6 +8,22 @@ sap.ui.define([
...
+	 * value assignments can be made. This is especially helpfull when it comes to own event handler implementations if the
```
Comment (danielheffter) [inline webapp/libs/api/valuation/CharacteristicGroupClientObject.js]: small typo: "helpful"

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js [hunk 4]
```diff
@@ -16,11 +32,28 @@ sap.ui.define([
 			this._sNamespace = sNamespace; 
 		},
 		
+		/**
+		 * Applies the given technical values to the Characteristic bound to the specified CharacteristicContextContainer control
+		 * @param {module:sap/i2d/lo/lib/vchclf/api/valuation/control/CharacteristicContextContainer} oCsticContextContainerControl - the CharacteristicContextContainer control for which values should be applied
+		 * @param {string[]} aTechnicalValues - the technical values as strings
```
Comment (danielheffter) [inline webapp/libs/api/valuation/CharacteristicGroupClientObject.js:37]: not sure if technical values are explained somewhere. In general, this is our internal naming of values which exist in the engine state and can be directly assigned without additional checks. Would it make a difference from your perspective to just name them values?

File: webapp/libs/api/valuation/control/CharacteristicContextContainer.js [hunk 5]
```diff
@@ -7,27 +7,81 @@ sap.ui.define([
...
+				 * @instance
+				 * @since 1.0.0
+				 * @public
+				 */
 				removeNoneValue: {
 					type: "boolean",
 					default: false
 				},
+				/**
+				 * Property - Whether domain values can be kept in case of a value assignment (performance optimization for independent Characteristics) - optional, default is <code>false</code>
```
Comment (danielheffter) [inline webapp/libs/api/valuation/control/CharacteristicContextContainer.js]: I would make it even clearer by saying domain values are only loaded once and not updated ... values can be kept can potentially be misunderstood how long (until next validation?)

File: webapp/libs/api/valuation/control/CharacteristicContextContainer.js [hunk 6]
```diff
@@ -7,27 +7,81 @@ sap.ui.define([
...
+				 * @type string
+				 * @instance
+				 * @since 1.0.0
+				 * @public
+				 */
 				name: {
 					type: "string"
 				},
+				/**
+				 * Property - Whether the container should load Domain Values - optional, default is <code>false</code>
```
Comment (danielheffter) [inline webapp/libs/api/valuation/control/CharacteristicContextContainer.js]: I would also add here: Whether the container should load Domain Values with the loading of characteristics. This is independent of value helps.

**Review by danielheffter — CHANGES_REQUESTED** [review]
In general, your internal user ID is now part of the files (as author). I guess this should be better removed.

File: doc/IMPLEMENTATION.md (hunk 1 above)
Comment (florianstephan01) [reply doc/IMPLEMENTATION.md]: fixed

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js (hunk 2 above)
Comment (florianstephan01) [reply webapp/libs/api/valuation/CharacteristicGroupClientObject.js]: fixed

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js (hunk 3 above)
Comment (florianstephan01) [reply webapp/libs/api/valuation/CharacteristicGroupClientObject.js]: fixed

File: webapp/libs/api/valuation/control/CharacteristicContextContainer.js (hunk 5 above)
Comment (florianstephan01) [reply webapp/libs/api/valuation/control/CharacteristicContextContainer.js]: fixed

File: webapp/libs/api/valuation/control/CharacteristicContextContainer.js (hunk 6 above)
Comment (florianstephan01) [reply webapp/libs/api/valuation/control/CharacteristicContextContainer.js]: fixed

**Comment by florianstephan01** [comment]
Removed the author tag

File: webapp/libs/api/valuation/CharacteristicGroupClientObject.js (hunk 4 above)
Comment (florianstephan01) [reply webapp/libs/api/valuation/CharacteristicGroupClientObject.js:37]: Leave technical values after discussion.

**Pull request merged**